src/topology/algebra/module/strong_topology.lean

```diff
@@ -224,6 +227,49 @@ protected lemma has_basis_nhds_zero [topological_space F]
     (λ SV, {f : E →SL[σ] F | ∀ x ∈ SV.1, f x ∈ SV.2}) :=
 continuous_linear_map.has_basis_nhds_zero_of_basis (𝓝 0).basis_sets
 
+variables (G) [topological_space F] [topological_space G]
+
+/-- Pre-composition by a *fixed* continuous linear map as a continuous linear map.
+Note that in non-normed space it is not always true that composition is continuous
+in both variables, so we have to fix one of them. -/
+@[simps] def precomp [topological_add_group G] [has_continuous_const_smul 𝕜₃ G]
+  [ring_hom_surjective σ] [ring_hom_isometric σ] (L : E →SL[σ] F) :
+  (F →SL[τ] G) →L[𝕜₃] (E →SL[ρ] G) :=
+{ to_fun := λ f, f.comp L,
+  map_add' := λ f g, add_comp f g L,
+  map_smul' := λ a f, smul_comp a f L,
+  cont :=
+  begin
+    letI : uniform_space G := topological_add_group.to_uniform_space G,
+    haveI : uniform_add_group G := topological_add_comm_group_is_uniform,
+    rw (strong_topology.embedding_coe_fn _ _ _).continuous_iff,
+    refine (uniform_on_fun.precomp_uniform_continuous _).continuous.comp
+      (strong_topology.embedding_coe_fn _ _ _).continuous,
+    exact λ S hS, hS.image L,
+  end }
```

Comment:
This should be a standalone lemma I think

Reply:
I would guess `(precomp G L).continuous` is enough ? In Lean 3 this usually works quite well, and usually we don't add this lemma (see e.g [continuous_linear_map.compSL](https://leanprover-community.github.io/mathlib_docs/analysis/normed_space/operator_norm.html#continuous_linear_map.compSL))
That said maybe we'll want to add it if Lean4 is a bit more rigid on unification.